app/components/pull-request-page.js

```diff
@@ -109,22 +109,25 @@ export default Ember.Component.extend({
           "standardSet.jurisdiction.title",
         ]
 
-        if (!_.includes(fieldsToDetermineValidate, errorObj[0].key)) {
-          set(this, "descriptionIsValid", true)
-        }
+        let allErrorKeys = _.map(errorArray, error => {
+          return error.key
+        })
+
+        let descriptionIsValid = _.size(_.intersection(fieldsToDetermineValidate, allErrorKeys)) === 0
+        set(this, "descriptionIsValid", descriptionIsValid)
 
         let formatValidateFields = ["indentation needed", "notation needed", "not enough standards"]
-        let checkFormat = _.intersection(errorObj[0].validation, formatValidateFields)
+        let standardsErrors = _.chain(errorArray)
+          .map(error => error.validation)
+          .flatten()
+          .intersectionBy(formatValidateFields)
+          .value()
```

Comment:
I'd change this variable name. standardsError seems vague and it sounds like it's more like `formattingErrors`. Is that right?